### PR TITLE
BigInt and BinaryReader/Writer inline helpers

### PR DESCRIPTION
**Pull request by pyramation**

### CODE REVIEW

File: packages/osmojs/src/codegen/capability/v1/capability.ts [hunk 1]
```diff
@@ -95,259 +86,203 @@ export interface CapabilityOwnersAminoMsg {
...
     let end = length === undefined ? reader.len : reader.pos + length;
     const message = createBaseCapability();
-
     while (reader.pos < end) {
       const tag = reader.uint32();
-
       switch (tag >>> 3) {
         case 1:
-          message.index = (reader.uint64() as Long);
+          message.index = BigInt(reader.uint64().toString());
```
Comment (webmaster128) [inline packages/osmojs/src/codegen/capability/v1/capability.ts:113]: Is thisString conversion needed here for the copy? What does `.uint64()` return?

**Review by webmaster128 — COMMENTED** [review]
👏 this is the way!

File: packages/osmojs/src/codegen/capability/v1/capability.ts (hunk 1 above)
Comment (Zetazzz) [reply packages/osmojs/src/codegen/capability/v1/capability.ts:113]: Good catch! I'll fix this right now! Really appreciate!

File: packages/osmojs/src/codegen/capability/v1/capability.ts [hunk 2]
```diff
@@ -95,259 +86,203 @@ export interface CapabilityOwnersAminoMsg {
...
  */
-
 export interface CapabilityOwnersSDKType {
   owners: OwnerSDKType[];
 }
-
 function createBaseCapability(): Capability {
   return {
-    index: Long.UZERO
+    index: BigInt("0")
```
[the remark this answers is not in the document]
Comment (Zetazzz) [reply packages/osmojs/src/codegen/capability/v1/capability.ts:94]: Ya, they're the same, but I'll unify them to BigInt(0)

**Pull request closed without merging**